### PR TITLE
Update service.py

### PR DESCRIPTION
**Pull request by bzlowrance**
##### ISSUE TYPE

- Bugfix Pull Request
##### COMPONENT NAME

Service/Restarted
##### ANSIBLE VERSION

```
<!--- Paste verbatim output from “ansible --version” between quotes -->


##### SUMMARY
<!--- Describe the change, including rationale and design decisions -->

<!---
Updated service module for AIX to allow arguments to be passed when state=restarted.
-->

```

```

```

### CODE REVIEW

**Comment by bcoca** [comment]
that is not a valid conditional

**Comment by bzlowrance** [comment]
Doh, yup missed the "or"
Sending update now.

Date: Tue, 26 Apr 2016 07:59:02 -0700

**Comment by bcoca** [comment]
you gave no reason for this PR, can you please fill out the template?

File: system/service.py [hunk 1]
```diff
@@ -1410,7 +1410,7 @@ def service_control(self):
             self.execute_command("%s -s %s" % (self.stopsrc_cmd, self.name))
             srccmd = self.startsrc_cmd
 
-        if self.arguments and self.action == 'start':
+        if self.arguments and (self.action == 'start' or self.action =='restart'):
```
Comment (jpic) [inline system/service.py:1413]: Parenthesis are not needed here

**Comment by bzlowrance** [comment]
Please update the code to correctly pass an argument to the AIX:startsrc command when starting or restarting via the service.py module.

**Pull request closed without merging**